src/shared/utils/util.ts

```diff
@@ -378,6 +378,11 @@ export class Util {
     return '***' + value.slice(value.length - visibleLength);
   }
 
+  static blankMail(value: string, visibleLength = 4): string {
+    const mailSplit = value.split('@');
+    return `${mailSplit[0].slice(0, visibleLength > value.length ? value.length : visibleLength)}***@${mailSplit[1]}`;
```

Comment:
I think the check `visibleLength > value.length ? value.length : visibleLength` is not required.